database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,13 +14,18 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
-            $table->id();
-            $table->string('name');
+            $table->increments('id');
```

Comment:
Why did you switch from `$table->id()` to `$table->increments('id')`?
`$table->id()` is just an alias of `$table->bigIncrements('id')`
https://laravel.com/docs/7.x/migrations#creating-columns